fbw-a32nx/src/base/flybywire-aircraft-a320-neo/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../../.eslintrc",
```

Comment:
Why not just put the rules in the root `.eslintrc` here? They are only meant for `base/` anyway.

Reply:
The A380 should not use any legacy JS.